File: payments/src/main/java/br/com/bank/payments/api/dto/events/RespostaCreditoEventDTO.java
package br.com.bank.payments.api.dto.events;

import br.com.bank.payments.domain.utils.enums.StatusResposta;


public class RespostaCreditoEventDTO {
    private String idTransacao;
    private StatusResposta status;
    private String mensagem;

    public RespostaCreditoEventDTO(String idTransacao, StatusResposta status, String mensagem) {
        this.idTransacao = idTransacao;
        this.status = status;
        this.mensagem = mensagem;
    }

    public RespostaCreditoEventDTO() {
    }

    public String getIdTransacao() {
        return idTransacao;
    }

    public void setIdTransacao(String idTransacao) {
        this.idTransacao = idTransacao;
    }

    public StatusResposta getStatus() {
        return status;
    }

    public void setStatus(StatusResposta status) {
        this.status = status;
    }

    public String getMensagem() {
        return mensagem;
    }

    public void setMensagem(String mensagem) {
        this.mensagem = mensagem;
    }
}

File: cards/src/main/kotlin/br/com/bank/cards/api/dto/events/RespostaCreditoEventDTO.kt
package br.com.bank.cards.api.dto.events

import br.com.bank.cards.domain.utils.enums.StatusResposta
import java.math.BigDecimal

data class RespostaCreditoEventDTO (
    val idTransacao: String,
    val idUsuario: String,
    val valor: BigDecimal,
    val status: StatusResposta,
    val mensagem: String
)